_publications/2019_memocode_dr.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: publication
+# The quotes make the : possible, otherwise you can do it without quotes
+title: "A Timeless Model for The Verification of Quasi-Periodic Distributed Systems"
+key: 2019_memocode_dr
+
+#YYYY_CONFSHORTNAME_PAPERSHORTNAME
+# paper | preprint | poster
+type: conference
+redirect_from: /publications/2019_memocode_dr
+
+# The shortname is used for auto-generated titels
+shortname: 2019_memocode_dr
+# add a 2:1 aspect ratio (e.g., width: 400px, height: 200px) to the folder /assets/images/papers/
+image: 2019_memocode_dr.png
+# 2018_tvcg_lineage.png
+# add a 2:1 aspect ratio teaser figure (e.g., width: 1200px, height: 600px) to the folder /assets/images/papers/
+image_large: 2019_memocode_dr_teaser.dr
+#2018_tvcg_lineage_teaser.png
+
+# Authors in the "database" can be used with just the key (lastname). Others can be written properly.
+authors:
+- Dabaghchian
+- Rakamaric
+
+journal-short: MEMOCODE
+year: 2019
+
+#bibentry: article | inproceedings | phdthesis | book
+
+bibentry: inproceedings
```

Comment:
You are missing a bunch of bibtex entries here.

Reply:
Done.